doc/manual/installation/uninstalling-nix.xml

```diff
@@ -0,0 +1,56 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      version="5.0"
+      xml:id="ch-uninstalling-nix">
+
+<title>Uninstalling Nix</title>
+
+<para>The following steps describes how to remove Nix from your system.</para>
+
+<para>First of all you need to remove the <filename>/nix</filename>
+directory:</para>
+
+<screen>sudo rm -rf /nix</screen>
```

Comment:
This and removing the sourcing of nix-profile.sh are really the only steps that are required.

Reply:
How does one remove the nix-profile.sh on osx, how do I locate it from the terminal (if that's what I am using)